docs/testplan/Smartswitch-test-plan.md

```diff
@@ -36,6 +47,19 @@ Purpose of the test is to verify smartswich platform related functionalities/fea
 For every test cases, all DPUs need to be powered on unless specified in any of the case.
 General convention of DPU0, DPU1, DPU2 and DPUX has been followed to represent DPU modules and the number of DPU modules can vary.
 
+## Testbed and Version
+
+The test runs on the os versions 202411 and above.
+Add a check to confirm that the test environment uses version 202411 or later; if the version is earlier, skip the test.
+After the above check, it needs to check DPUs in the testbed are in dark mode or not. 
+If it is in dark mode, then power up all the DPUs. 
+Dark mode is one in which all the DPUs admin_status are down. 
+
+## Topology
+
+New topology called smartswitch-t1 has been added for running smartswitch cases.
+T1 cases also runs on the new topology. 
```

Comment:
Do we run 'any' topo test cases also on smartswitch?

Reply:
Yes it should.